judge/judgedaemon.main.php

```diff
@@ -219,7 +219,7 @@ function rest_encode_file(string $file, $sizelimit = true) : string
     } else {
         error("Invalid argument sizelimit = '$sizelimit' specified.");
     }
-    return urlencode(base64_encode(dj_file_get_contents($file, $maxsize)));
```

Comment:
The old code did:
```
'batch=' . json_encode($unsent_judging_runs),
```
which i think made it such that it *needed* url encoding. If that would have been `['batch'=>json_encode($unsent_judging_runs)]` it would also break.